projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -766,7 +766,7 @@ Once saved in the right place, either restart NVDA or choose Reload Plugins foun
     		focusObj = api.getFocusObject()
     		name = focusObj.name
     		windowClassName = focusObj.windowClassName
-    		ui.message("class for %s window: %s" % (name, windowClassName))
+    		ui.message(f"class for {name} window: {windowClassName}")
```

Comment:
can you please reformat this entire code block:

- remove the space indentation
- wrap the block with 3 back ticks \`\`\`py \`\`\`
- remove the start and end lines

~~~suggestion
```py
		ui.message(f"class for {name} window: {windowClassName}")
```
~~~